Sources/ComposableArchitecture/Documentation.docc/Tutorials/BuildingSyncUps/06-SyncUpDetail/TestingSyncUpDetail-02-code-0004.swift

```diff
@@ -11,20 +11,19 @@ struct SyncUpDetailTests {
       id: SyncUp.ID(),
       title: "Point-Free Morning Sync"
     )
+    
+    @Shared(.syncUps) var syncUps = [syncUp]
+
```

Comment:
📓 This new step explain how to test that the list of sync-ups has been updated by first adding a ref to the shared list...